src/mlagility/cli/cli.py

```diff
@@ -296,6 +308,35 @@ def main():
         default=filesystem.DEFAULT_CACHE_DIR,
     )
 
+    #######################################
+    # Parser for the "state" command
+    #######################################
+
+    state_parser = subparsers.add_parser(
+        "state", help="Print the state of a build in a target cache"
```

Comment:
> Print the state of a build in a target cache

I really like the description of what this feature does. However, it is not printing the state of **a** build now, but it is printing the state of **one or more builds**. Rather than changing the description to match the functionality, can we change the functionality to match the description? We will never need to print the state of more than one build at a time anyways.

This also allows us to get rid of the `-b` flag and make the `build_name` a positional arg.

Reply:
Please also make `build_name` positional on `delete` for consistency

Reply:
Done!